backend/de.metas.marketing/base/src/main/java/de/metas/marketing/base/interceptor/AD_User_MKTG_Channels.java

```diff
@@ -26,36 +26,55 @@
 import de.metas.marketing.base.api.IMKTGChannelDao;
 import de.metas.marketing.base.model.I_AD_User_MKTG_Channels;
 import de.metas.user.UserId;
+import de.metas.user.api.IUserDAO;
 import de.metas.util.Services;
 import lombok.NonNull;
-import org.adempiere.ad.callout.annotations.Callout;
 import org.adempiere.ad.modelvalidator.annotations.Interceptor;
 import org.adempiere.ad.modelvalidator.annotations.ModelChange;
 import org.adempiere.exceptions.AdempiereException;
+import org.adempiere.service.ISysConfigBL;
 import org.compiere.model.ModelValidator;
+import org.compiere.util.Env;
 
-@Callout(I_AD_User_MKTG_Channels.class)
 @Interceptor(I_AD_User_MKTG_Channels.class)
 public class AD_User_MKTG_Channels
 {
 	public static final AD_User_MKTG_Channels INSTANCE = new AD_User_MKTG_Channels();
 
 	private final IMKTGChannelDao mktgChannelDao = Services.get(IMKTGChannelDao.class);
+	private final IUserDAO userDAO = Services.get(IUserDAO.class);
+	private final ISysConfigBL sysConfigBL = Services.get(ISysConfigBL.class);
 
 	private static final AdMessageKey MSG_CAN_NOT_REMOVE_CHANNEL = AdMessageKey.of("de.metas.marketing.base.userMarketingChannelRemovalError");
+	private static final String SYS_CONFIG_MARKETING_CHANNELS_ENFORCED = "de.metas.marketing.EnforceUserHasMarketingChannels";
 
 	private AD_User_MKTG_Channels()
 	{
 	}
 
 	@ModelChange(
 			timings = { ModelValidator.TYPE_BEFORE_DELETE })
-	public void checkIfCanBeDeleted(@NonNull final I_AD_User_MKTG_Channels userMktgChannel)
+	public void checkIfCanBeDeleted(@NonNull final I_AD_User_MKTG_Channels userMktgChannels)
 	{
-		final int count = mktgChannelDao.retrieveMarketingChannelsCountForUser(UserId.ofRepoId(userMktgChannel.getAD_User_ID()));
+		if (!isMarketingChannelsUseEnforced())
+		{
+			return;
+		}
+
+		if (userDAO.isSystemUser(UserId.ofRepoId(userMktgChannels.getAD_User_ID())))
+		{
+			return;
+		}
+
+		final int count = mktgChannelDao.retrieveMarketingChannelsCountForUser(UserId.ofRepoId(userMktgChannels.getAD_User_ID()));
 		if (count == 1)
 		{
 			throw new AdempiereException(MSG_CAN_NOT_REMOVE_CHANNEL).markAsUserValidationError();
 		}
 	}
+
+	private boolean isMarketingChannelsUseEnforced()
+	{
+		return sysConfigBL.getBooleanValue(SYS_CONFIG_MARKETING_CHANNELS_ENFORCED, false, Env.getAD_Client_ID(Env.getCtx()), Env.getAD_Org_ID(Env.getCtx()));
```

Comment:
please check this with the client-id and org-id of the I_AD_User in question. (same as in the tab-callout)